hack/verify-import-boss.sh

```diff
@@ -22,4 +22,7 @@ if ! which import-boss > /dev/null; then
   exit 1
 fi
 
-import-boss -i k8s.io/cloud-provider-openstack/... --verify-only
+# prepare the vendor dir
+go mod vendor
```

Comment:
why using  `go mod vendor` here??

Reply:
because it required by `import-boss` and only during tests. I don't know what is the purpose of using the `import-boss`